react/react1/week3/todo-app-week3/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import Header from './components/Header';
+import Timer from './components/Timer';
+import TodoList from './components/TodoList';
+import Form from './components/Form';
+import './App.css';
+
+function App() {
```

Comment:
Your App component has another view. 
All other components look like this:
const ComponentName = ()=> {}